ideascube/models.py

```diff
@@ -109,6 +109,10 @@ def has_module_perms(self, app_label):
         return True
 
     @classmethod
+    def get_system_user(cls):
+        return cls.objects.get(serial='__system__')
```

Comment:
This won't work, because the default queryset (see #430) completely hides the system user.

In addition, this should probably be a manager method?